pkg/cmd/serviceaccount/create/create.go

```diff
@@ -154,7 +149,8 @@ func runCreate(opts *options) error {
 	creds := &credentials.Credentials{
 		ClientID:     serviceacct.GetClientId(),
 		ClientSecret: serviceacct.GetClientSecret(),
-		TokenURL:     cfg.MasAuthURL + "/protocol/openid-connect/token",
+		// TODO new location of the token url? From where we can take this value?
```

Comment:
CC @ziccardi @Rajagopalan-Ranganathan 

We need to discuss how CLI/UI will get new auth url:

What API will return it (maybe we can add that earlier than migration for end to end testing using still old value

Reply:
CC @akoserwal